code/datums/status_effects/debuffs.dm

```diff
@@ -967,3 +967,81 @@
 /datum/status_effect/cloudstruck/Destroy()
 	. = ..()
 	QDEL_NULL(mob_overlay)
+
+//Deals with ants covering someone.
+/datum/status_effect/ants
+	id = "ants"
+	tick_interval = 1 SECONDS
+	status_type = STATUS_EFFECT_REFRESH
+	alert_type = /atom/movable/screen/alert/status_effect/ants
+	duration = 5 MINUTES //Keeping the normal timer makes sure people can't somehow dump 500+ ants on someone so they stay there for like 30 minutes
+	examine_text = "<span class='warning'>SUBJECTPRONOUN is covered in ants!</span>"
+	var/ants_remaining = 0 //Will act as the main timer as well as changing how much damage the ants do.
+
+/datum/status_effect/ants/on_creation(mob/living/new_owner, amount_left)
+	if(isnum(amount_left) && ants_remaining == 0)
+		to_chat(new_owner, "<span class='userdanger'>You're covered in ants!</span>")
+		ants_remaining = amount_left
+		RegisterSignal(new_owner, COMSIG_COMPONENT_CLEAN_ACT, .proc/ants_washed)
+	else if(ants_remaining >= 0)
+		to_chat(new_owner, "<span class='userdanger'>You're covered in MORE ants!</span>")
+		if(prob(1)) //Extremely rare
+			new_owner.say("THIS SITUATION HAS BEEN MADE WORSE BY THE ADDITION OF YET MORE ANTS!!", forced = /datum/status_effect/ants)
+		ants_remaining += amount_left
+	. = ..()
+
+/datum/status_effect/ants/on_remove()
+	ants_remaining = 0
+	to_chat(owner, "<span class='notice'>All of the ants are off of your body!</span>")
+	UnregisterSignal(owner, COMSIG_COMPONENT_CLEAN_ACT, .proc/ants_washed)
+	. = ..()
+
+/datum/status_effect/ants/proc/ants_washed()
+	SIGNAL_HANDLER
+	owner.remove_status_effect(STATUS_EFFECT_ANTS)
+	return COMPONENT_CLEANED
+
+/datum/status_effect/ants/tick(delta_time)
+	var/mob/living/carbon/human/victim = owner
+	victim.adjustBruteLoss(max(0.2, round((ants_remaining * 0.008),0.1))) //Roughly 20 brute over 50 seconds.
```

Comment:
That's pretty brutal, death in like 400 seconds or something right?

Reply:
Oh and this means it'll do at least 0.2 damage per tick, is that what you intended? max and min can be kinda fuzzy to think about

Reply:
It's meant to do at least 0.2, but if that's too much I can lower that down to 0.1 or something. I don't _think_ 20 brute over 50 seconds is too bad, since that damage is lowered over time, however I'm always okay with balancing things better.

Reply:
it's lowered over time?

Reply:
Yes, the damage scales with the remaining ants on the person's body.

Reply:
right but you're always doing at least 0.2 damage per tick, which I assume? is where the 20 brute over 50 seconds comes from.

Of course now that I think about the fact that the ants come off as time goes on I'm less worried

Reply:
I lowered it from 0.2 to 0.1 now either way, to lower the damage if someone somehow has both ingested ants and is covered in ants